tests/sync/session.cpp

```diff
@@ -161,6 +165,38 @@ TEST_CASE("SyncSession: management by SyncUser", "[sync]") {
         EventLoop::main().run_until([&] { return session_is_active(*session1) && session_is_active(*session2); });
         REQUIRE(user->all_sessions().size() == 2);
     }
+
+    SECTION("a SyncUser properly allows registration of sessions that were recreated") {
+        const std::string user_id = "user1e";
```

Comment:
This appears to only be used the once. Doesn't seem worth having a local variable.